rdkit/Chem/Draw/MolDrawing.py

```diff
@@ -250,54 +230,48 @@ def _drawBond(self, bond, atom, nbr, pos, nbrPos, conf, width=None, color=None,
           wcolor = color
         else:
           p2, p1 = newpos, newnbrPos
-          if color2 is not None:
-            wcolor = color2
-          else:
-            wcolor = self.drawingOptions.defaultColor
+          wcolor = color2
         if bDir == Chem.BondDir.BEGINWEDGE:
           self._drawWedgedBond(bond, p1, p2, color=wcolor, width=width)
         elif bDir == Chem.BondDir.BEGINDASH:
           self._drawWedgedBond(bond, p1, p2, color=wcolor, width=width,
                                dash=self.drawingOptions.dash)
-
       else:
-        self.canvas.addCanvasLine(newpos, newnbrPos, linewidth=width, color=color, color2=color2)
+        addDefaultLine(newpos, newnbrPos)
```

Comment:
I don't think the use of partial adds anything here.  I tend to use currying/partial application when I don't want to expose things outside the API to a consumer.  I don't think that really applies in this situation.

That being said, this is a style comment and nothing I'm overly concerned about.